docs/MAVLINK_PARAMETERS.md

```diff
@@ -0,0 +1,32 @@
+
+This MAVLink component is available only under certain conditions:
+- in the Tx module the setting "Tx Mav Component" must be set to "enabled". This is usually the default setting. If not, when it must be configured using one of the other methods (CLI, Lua script, OLED). 
+- a receiver is connected for which the setting "Rx Ser Link Mode" is configured to "mavlink" or "mavlinkX".
+- if no receiver is connected then the MAVLink component is available if the last receiver which the Tx module "remembers" was configured with "Rx Ser Link Mode" = "mavlink" or "mavlinkX". This is usually the last receiver which was connected to the Tx module and for which a parameter store operation was done. 
+
+As for the other methods, if a receiver is connected, then the parameters of both the Tx module and the receiver can be configured. If no receiver is connected, then only the parameters of the Tx module can be configured. Note however that the receiver parameters are nevertheless shown.
+
+The MAVLink parameters are identical to those described in [Configuration Parameters](PARAMETERS.md), but have different names which conform to the MAVLink standard. They should be easy to recognize however, for instance, the parameter "Rx Ser Link Mode" shows up as "RX_SER_LNK_MODE". Four exceptions exist:
+- the bind phrase is presented as an uint32_t number instead of a string of 6 chars as in the other methods, and the parameter is called "BIND_PHRASE_U32" (setting a bind phrase is thus a bit inconvenient, but the MAVLink parameter system does not allow us to do better).
+- the parameter "TX_MAV_COMPONENT" (= "Tx Mav Component") cannot be set or modified and will always be 1. This is to prevent unintentional disabling of the MAVLink component.
+- the parameter "RX_SER_LNK_MODE" (= "Rx Ser Link Mode") cannot be set to 0. This is to prevent unintentional disabling of the MAVLink component.
+- the additional parameter "PSTORE" is available. Parameter changes are not permanently stored in the Tx module or receiver when written. In order to store them permanently, the parameter PSTORE must be set to 1 and written to the component. This is somewhat similar to the CLI's pstore command or the Lua script's "Store" button. 
+
+Some GCSes, such as MissionPlanner, do not update the parameter values to the actual values taken by the component upon a write operation. This can result in situations where the parameter values shown in the GCS are invalid, i.e., do not reflect the setting in the Tx module or reciever (the available parameter settings can be specific to a particular mLRS system). It is thus recommended practice to do a parameter refresh after parameter writes, and to double check the resulting setting before a PSTORE.
```

Comment:
'Some GCS software...'

Perhaps it makes sense to provide an example set of steps, something like:

1. Change the value of the parameter you wish to update
2. Click Write, to temporarily write the parameter
3. Click  Refresh, to check that the parameter was updated
4. Set PSTORE to 1
5. Click Write to permanently store the parameter